services/QuillLMS/app/controllers/teachers/classroom_manager_controller.rb

```diff
@@ -244,6 +244,12 @@ def unset_view_demo
     redirect_to '/profile'
   end
 
+  # rubocop:disable Naming/PredicateName
+  def is_demo
```

Comment:
I'm kind of late to the party here, but can you maybe say a little bit about what the difference between `view_demo` and `is_demo` are?  (My gut says this method/label should be `demo` and this is kind of what rubocop is advocating as well)